dockerhub.docker-compose.yml

```
@@ -1,4 +1,3 @@
-version: "3"
 services:
```

```suggestion
services:
```
Is the version key deprecated? I noticed you have removed it.

---

Yes I was getting warnings. See https://docs.docker.com/reference/compose-file/version-and-name/

---

thanks!